packages/common/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@lmc-eu/spirit-common",
+  "version": "0.0.0",
```

Comment:
Isn't it 0.1.0 for initial package versions?

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
This will be updated by Lerna when determining the version for release, e.g. Lerna will take care of this.

If I use `0.1.0`, the Lerna will update this to `0.2.0` and it will be the first version published.